src/shared/components/CountdownTimer/CountdownTimer.tsx

```diff
@@ -28,6 +28,10 @@ export const CountdownTimer = ({ launchDate }: CountdownTimerProps) => {
     return () => timerCleanup(timeoutId);
   });
 
+  useEffect(() => {
+    setTimerValues(calculateTimerValues(timeToLaunch));
```

Comment:
Why don’t you place the function in the previous hook?

Reply:
I want to call setTimerValues once, after component mounting. If I call it in the previous hook it will be an infinite loop because I change the state in this function.